celesta-test/src/test/java/ru/curs/celesta/dbutils/adaptors/AbstractAdaptorTest.java

```diff
@@ -731,7 +730,7 @@ public void testColumnInfoForMaterializedView()
 
 
     @Test
-    public void updateColumn() throws CelestaException, ParseException, IOException, SQLException {
+    public void updateColumn() throws ParseException, IOException, SQLException {
```

Comment:
Codacy says this method is 'really long', what does it count? if it counts lines, then I believe at this point we can ignore. If it counts statements, then wrapping assertions into assertAll blocks will do. Anyway using assertAll is good

Reply:
Codacy doesn't explain this exactly. AssertAll requires the using of final variables, which is inconvenient in our case. I think, that we can ignore this.